mne/epochs.py

```diff
@@ -1810,6 +1811,69 @@ def save(self, fname, split_size='2GB', fmt='single', overwrite=False,
             this_epochs.event_id = self.event_id
             _save_split(this_epochs, fname, part_idx, n_parts, fmt)
 
+    def export(self, fname, fmt="auto"):
+        """Export Epochs to external formats.
+
+        Supported formats: EEGLAB (set, uses :mod:`eeglabio`)
+
+        .. warning::
+            Since we are exporting to external formats, there's no
+            guarantee that all the info will be preserved in the external
+            format. To save in native MNE format (fif) without info loss,
+            use :func:`save` instead.
+
+        Parameters
+        ----------
+        fname : str
+            Name of the export file.
+        fmt : str
+            Format of the export. Defaults to auto, which will infer
+            format from fname's extension.
+
+        Notes
+        -----
+        For EEGLAB format, channel locations are expanded to full EEGLAB
+        format. For more details see :func:`eeglabio.utils.cart_to_eeglab`.
+        """
+        if fmt == "auto":
+            fmt = op.splitext(fname)[1]
```

Comment:
Your'e using double-quotes here and single-quotes in other places. Can you please make this consistent? (My personal preference being single-quotes :))